Sources/AppStoreConnectCLI/Commands/TestFlight/Sync/TestFlightPullCommand.swift

```diff
@@ -28,13 +28,8 @@ struct TestFlightPullCommand: CommonParsableCommand {
     func run() throws {
         let service = try makeService()
 
-        // TODO: A new service function should be created to efficiently gather these models
-        let apps = try service.listApps(bundleIds: filterBundleIds)
-        let identifiers = apps.map { app in AppLookupIdentifier.appId(app.id) }
-        let testers = try service.listBetaTesters(filterIdentifiers: identifiers, limit: 200)
-        let groups = try service.listBetaGroups(filterIdentifiers: identifiers)
+        let testflightProgram = try service.getTestFlightProgram()
```

Comment:
I can see `filterBundleIds` is no longer get used, we might want to remove this 
```
//    @Option(
//        parsing: .upToNextOption,
//        help: "Filter by only including apps with the specified bundleIds in the configuration"
//    ) var filterBundleIds: [String]
```
Or are we going to use this in the future?

Reply:
That was a mistake! Thanks for catching it 👍